drawbicyc/include/drawbicyc/cases/Lasher_Trapp/plots.hpp

```diff
@@ -2,26 +2,34 @@
 
 
 const std::vector<std::string> series_Lasher_Trapp({
-  "clfrac",
+  //"clfrac",
   "lwp",
   "rwp",
+  "cwp",
+  "lwm",
+  "cwm",
+  "rwm",
   "surf_precip",
   "acc_precip",
-  "cl_nc",
+  "acc_vol_precip",
+  //"cl_nc",
   "cloud_base",
-  "surf_flux_latent",
-  "surf_flux_sensible",
-  "sd_conc_avg",
+  "RH_max",
+  "cloud_top_height",
+  "total_droplets_number",
+  //"surf_flux_latent",
+  //"surf_flux_sensible",
+  //"sd_conc_avg",
   //"mass_dry",
-  "cl_gccn_conc",
-  "gccn_conc",
-  "cl_non_gccn_conc",
-  "non_gccn_conc",
-  "cl_gccn_to_non_gccn_conc_ratio"
+  //"cl_gccn_conc",
+  //"gccn_conc",
+  //"cl_non_gccn_conc",
+  //"non_gccn_conc",
+  //"cl_gccn_to_non_gccn_conc_ratio"
   //, "cl_gccn_meanr"
   //,"cl_avg_cloud_rad"
   // "sd_conc_std_dev",
-  // // "tot_water"
+ // "tot_water"
```

Comment:
Again, make a new case ICMW2020_cumulus_congestus instead of modifying the Lasher_Trapp case.